### DISCUSSION TITLE
Choice of MPC ECDSA protocol

### DISCUSSION THREAD

**Discussion by matthiasgeihs** [near/mpc-recovery#899] [Q&A]
Hey, cool project! Saw that you are using `cait-sith` threshold ECDSA under the hood. Since this is a relatively uncommon protocol (afaik), was wondering what lead you to the choice if this protocol over other protocols like `CGGMP` or `DKLS`. Cheers!

**Comment by volovyks** [comment]
Hi @matthiasgeihs !
At the time it had a more solid Rust implementation.
